src/contextmenu-layer.js

```diff
@@ -59,11 +60,19 @@ export default function(identifier, configure) {
                     }
                 });
             },
-            render() {
-                return React.createElement(this.props.renderTag, {
-                    className: "react-context-menu-wrapper",
-                    onContextMenu: this.handleContextClick
-                }, React.createElement(Component, this.props));
+            render: function() {
+
+                var attributes = this.props.attributes;
+                if(attributes == undefined) attributes = {};
```

Comment:
Oh I completely missed that. I updated the contextmenu-layer and menu-item to do use the default props.